fractal.sdk/src/main/java/com/yanchware/fractal/sdk/Automaton.java

```diff
@@ -97,6 +100,15 @@ private static void createOrUpdateBlueprint(LiveSystem liveSystem) throws Instan
         blueprintService.createOrUpdateBlueprint(blueprintCommand, liveSystem.getFractalId());
     }
 
+    public static void instantiate(Environment environment) throws InstantiatorException {
```

Comment:
I think we should add some documentation on these public methods to provide guidance to the developers